src/sentry/static/sentry/app/components/sentryDocumentTitle.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {FunctionComponent} from 'react';
+import DocumentTitle from 'react-document-title';
+
+
+type DocumentTitleProps = {
+    title: string;
+    objSlug: string;
```

Comment:
Should we name this something a bit more generic? It's *generally* an org slug, but I could see it being `<org> - <project>` where it provides more context.

We could also leave as-is, but at the minimum a comment about what this field is would be great.

Reply:
@billyvg `objSlug`  because I use it in Project Settings either :) thats why objSlug

title for project as I remember `<page_title> - <project_slug> - Sentry` described in Issue

Reply:
yeah, I think just a comment about what to put here would be good then